windows/ReactTestApp/MainPage.h

```diff
@@ -12,15 +13,26 @@ namespace winrt::ReactTestApp::implementation
                               Windows::UI::Xaml::RoutedEventArgs);
         void LoadFromDevServer(Windows::Foundation::IInspectable const &,
                                Windows::UI::Xaml::RoutedEventArgs);
+        void MainPage::OpenReactMenu(Windows::Foundation::IInspectable const &,
+                                     winrt::Windows::UI::Xaml::RoutedEventArgs);
         Windows::Foundation::IAsyncAction
         OnNavigatedTo(Windows::UI::Xaml::Navigation::NavigationEventArgs const &e);
         using Base = MainPageT;
 
+        void OnItemClick(Windows::Foundation::IInspectable const &sender,
+                         Windows::UI::Xaml::Controls::ItemClickEventArgs e);
+
     private:
         ::ReactTestApp::ReactInstance reactInstance_;
 
         void SetReactComponentName(Windows::Foundation::IInspectable const &,
-                                   Windows::UI::Xaml::RoutedEventArgs e);
+                                   Windows::UI::Xaml::RoutedEventArgs);
+        Windows::UI::Xaml::Controls::Button
+        GetComponentMenuButton(::ReactTestApp::Component &component);
+        void CoreTitleBarLayoutMetricsChanged(
+            winrt::Windows::ApplicationModel::Core::CoreApplicationViewTitleBar const &sender,
```

Comment:
Nit: Unnecessary namespace:
```suggestion
            Windows::ApplicationModel::Core::CoreApplicationViewTitleBar const &sender,
```